buildspec_publish_to_sar.yml

```diff
@@ -6,17 +6,21 @@ phases:
       java: openjdk11
     commands:
       - pip3 install aws-sam-cli
+      - apt install -q -y jq
 
   build:
     commands:
       # Use AWS SAM to build and package the application by using AWS CloudFormation
       - sam build
-      - sam package --template-file .aws-sam/build/template.yaml --s3-bucket $S3_BUCKET --output-template-file sampackaged.yaml --force-upload
+      - sam package --template-file .aws-sam/build/template.yaml --s3-bucket $S3_BUCKET --output-template-file sampackaged_raw.yaml
   post_build:
     commands:
+      # Getting latest version tag from git
+      - APPLICATION_VERSION=$(curl -s  https://api.github.com/repos/BIBSYSDEV/$GIT_REPO/releases/latest | jq -r '.tag_name')
+      # Updating metadata.labels in template
+      - envsubst '${CODEBUILD_RESOLVED_SOURCE_VERSION},${GIT_REPO}' < sampackaged_raw.yaml > sampackaged.yaml
```

Comment:
These are both provided by the CodeBuild instance?

Reply:
CODEBUILD_RESOLVED_SOURCE_VERSION is set by codebuild, GIT_REPO is manually set from the build codepipeline as environment variable.